packages/cubejs-teradata-jdbc-driver/dist/src/TeradataDriver.d.ts

```diff
@@ -0,0 +1,30 @@
+import { JDBCDriver, JDBCDriverConfiguration } from '@cubejs-backend/jdbc-driver';
```

Comment:
Remove the entire dist/ folder from the commit.

Reply:
it's in the .gitignore now